src/content/reference/fleet-management/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Fleet Management
+description: List of documents related to the management of developer environments, applications, and configurations across different environments.
```

Comment:
What exactly will we put here (under "Reference")?

And the usual removal of fill words / title case:

```suggestion
title: Fleet management
description: Management of developer environments, applications, and configurations across different environments.
```